Source/courses.py

```diff
@@ -0,0 +1,75 @@
+# Author: Isa Blancett
+# Project: Interactive Programming
+# Date: 10.30.2017
+# Description: Class for reading in a csv file containing course information and
+# creating and instance for each course
+# Acknowledgements: Script structure is modeled after Transcriptase's Game
+# repository on Git
+
+import csv
+
+class Courses(object):
+    """ class for the different course options the user can choose from
+
+    contains:
```

Comment:
Good job explaining what all of these are. An extra step you could do is specifically write what type these will end up being (ie: string, int, etc..). It's not necessary for this project, but it's good practice when you're making data structures like these.